bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/internal/TelegramHandler.java

```diff
@@ -196,7 +183,34 @@ public void initialize() {
         updateStatus(ThingStatus.UNKNOWN);
         delayThingOnlineStatus();
         TelegramBot localBot = bot = new TelegramBot.Builder(botToken).okHttpClient(botLibClient).build();
-        localBot.setUpdatesListener(this::handleUpdates, this::handleExceptions);
+        localBot.setUpdatesListener(this::handleUpdates, this::handleExceptions,
+                getGetUpdatesRequest(config.getLongPollingTime()));
+    }
+
+    private void createReceiverChatIdsAndAuthorizedSenderChatIds(List<String> chatIds) {
```

Comment:
It's a good idea to move code blocks to a new separate methods. The only thing that I don't like now is that this method has side effects (adding elements to a list which is a class members). Whereas for the initialize method it is quite clear that it's only called once or if it is called more often, the lists are cleared beforehand. Now, you need a very implicit knowledge on that new method at what point in time this method shall be called and how often.
The best way would be to remove the side effect, make it static and to just return the result. However, returning two lists is also a bit ugly (e.g. using the Pair<> class).
Or maybe we should at least move the `authorizedSenderChatId.clear(); receiverChatId.clear();` to that method too so that the result of the method is always the same (no matter when and how often it is called). What is your opinion?